src/js/setup-text-tracks.js

```diff
@@ -58,11 +64,10 @@ function attachDashTextTracksToVideojs(player, tech, tracks) {
       if (textTrack.mode === 'showing') {
         // Find the dash track we want to use
 
-        const dictionaryLookupResult = trackDictionary.find(
-          /* jshint loopfunc: true */
-          ({textTrack: dictionaryTextTrack}) => dictionaryTextTrack === textTrack
-          /* jshint loopfunc: false */
-        );
+        /* jshint loopfunc: true */
+        const dictionaryLookupResult = find(trackDictionary,
+          ({textTrack: dictionaryTextTrack}) => dictionaryTextTrack === textTrack);
```

Comment:
My feeling is that `(track) => track.textTrack === textTrack` is both shorter and easier to read.